src/index.js

```diff
@@ -2,16 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(<App/>, document.getElementById('root'));
```

Comment:
Wonder if we should add back the strictmode now that we solved the bug? 
Maybe it is not important for this project. Worth researching to figure out what that does.